src/sentry_core.c

```diff
@@ -303,6 +314,8 @@ sentry_options_new(void)
     memset(opts, 0, sizeof(sentry_options_t));
     opts->database_path = sentry__path_from_str(".sentry-native");
     sentry_options_set_dsn(opts, getenv("SENTRY_DSN"));
+    const char *debug = getenv("SENTRY_DEBUG");
+    opts->debug = debug && strlen(debug) > 0;
```

Comment:
Can we check for `"1"` explicitly? It's quite common to disable by setting `SENTRY_DEBUG=0`.